torch/_dynamo/eval_frame.py

```diff
@@ -322,15 +382,19 @@ def __enter__(self):
         self.backend_cache_manager.__enter__()
         self.backend_ctx = self.extra_ctx_ctor()
         self.backend_ctx.__enter__()
-        self.dynamic_ctx = enable_dynamic(self.dynamic, self.export)
-        self.dynamic_ctx.__enter__()
```

Comment:
This look suspicious

Reply:
why?

Reply:
We're no longer forcing dynamic to be enabled/disabled based on the setting of `dynamic=` on torch.compile. I think Jon's intention was to get it done some other way, but IDK, with a close SEV-minded reread, some parts seem dodgy.